docs/portal.properties-Reference.md

```diff
@@ -308,9 +308,8 @@ ehcache.static_repository_cache_one.max_mega_bytes_heap=
 ehcache.static_repository_cache_one.max_mega_bytes_local_disk=
 ```
 
-Additional properties can be specified for cache statistics monitoring. To log metrics regarding memory usage, set `ehcache.statistics_enabled` to true. Logged metrics and additional information such as cache size and cached keys are available through an optional endpoint. The optional endpoint is turned off by default but can be turned on by setting `cache.statistics_endpoint_enabled` to true. 
+Logged metrics and additional information such as cache size and cached keys are available through an optional endpoint. The optional endpoint is turned off by default but can be turned on by setting `cache.statistics_endpoint_enabled` to true.
```

Comment:
This info should be removed since we are no longer supporting this property.

Reply:
@ao508 We are supporting `cache.statistics_endpoint_enabled`.  We are not supporting `ehcache.statistics_enabled`.  Confusing I know.  The endpoint one is to say whether or not we want the endpoints related to viewing various information about the caches.  The `ehcache.statistics_enabled` I thought was meant to turn on/off `EhCacheStatistics.java` and/or the logging that starts with "CACHE_STATISTICS START", but it doesn't seem to do either of those things.